Sources/AnimalKingdomAPI/graphql/AnimalSchema.graphqls

```diff
@@ -1,6 +1,6 @@
 type Query {
   allAnimals: [Animal!]!
-  classroomPets: [ClassroomPet!]!
+  classroomPets: [ClassroomPet]
```

Comment:
I also removed the Non-Nulls from classroomPets to give me something modify in tests. Hope that's okay.